website/pages/docs/cheatsheet.mdx

```diff
@@ -0,0 +1,124 @@
+import { Callout } from "nextra-theme-docs";
+
+# Cheat sheet
+
+Get started with Scarb by following a cheatsheet of the most important commands.
+
+## Creating new project
+
+```shell copy
+scarb new project/directory
+```
+
+<Callout>
+  There is `scarb init{:shell}` which runs in current directory instead of creating new
+  one.
+</Callout>
+
+## Compiling
+
+```shell copy
+scarb build
+```
+
+Built artifacts will be written to `target/release` directory.
+
+<Callout>`scarb clean{:shell}` cleans `target` directory.</Callout>
+
+### Building CASM
+
+Add following to `Scarb.toml`:
+
+```toml copy
+[lib]
+casm = true
+```
+
+Adding following line to `[lib]{:toml}` section will stop building Sierra code:
+
+```toml copy
+sierra = false
+```
+
+### Building StarkNet contracts
+
+Add following to `Scarb.toml`:
+
+```toml copy
+[[target.starknet-contract]]
+```
+
+<Callout type="warning">
+  Ensure there is no `[lib]{:toml}` section, they are conflicting until
+  [#91](https://github.com/software-mansion/scarb/issues/91) will be done.
+</Callout>
+
+## Adding dependencies
+
+### In manifest
+
+Add dependency hosted on a Git repository:
+
+```toml copy
+[dependencies]
+quaireaux = { git = "https://github.com/keep-starknet-strange/quaireaux.git" }
+```
+
+Add dependency located in local path:
+
+```toml copy
+[dependencies]
+quaireaux = { path = "path/to/quaireaux" }
+```
+
+<Callout type="warning">
+  You can add `version` field to specify package version, but this will do more
```

Comment:
I think this should be also put in "Specifying Dependencies" page

Reply:
This was actually misleading! Specifying version is yet OK, fixing git reference is not